packages/adapters/wagmi/src/tests/client.test.ts

```diff
@@ -106,12 +106,14 @@ describe('WagmiAdapter', () => {
 
     it('should set wagmi connectors', async () => {
       vi.spyOn(wagmiCore, 'watchConnectors').mockImplementation(vi.fn())
+      vi.spyOn(wagmiCore, 'watchConnectors')
 
       await adapter.syncConnectors(
         { networks: [mainnet], projectId: 'YOUR_PROJECT_ID' },
         mockAppKit
       )
 
+      expect(wagmiCore.watchConnectors).toHaveBeenCalled()
```

Comment:
Let's check it with `toHaveBeenCalledOnce()`? to double check we don't overcall this?